src/NUnitFramework/framework/SpecialValue.cs

```diff
@@ -33,7 +33,7 @@ public enum SpecialValue
     {
         /// <summary>
         /// Null represents a null value, which cannot be used as an 
-        /// argument to an attriute under .NET 1.x
+        /// argument to an attribute under .NET 1.x
```

Comment:
By the way, @jnm2 - I forgot to mention how happy this PR makes my inner-perfectionist. 😄 Nice one!